recipes/README.md

```diff
@@ -0,0 +1,3 @@
+# Welcome to `staged-recipes`!
+
+Looking to submit a recipe? Check out [the sandbox](https://mybinder.org/v2/gh/pangeo-forge/sandbox/binder). It has a template for the `recipe.py` and `meta.yml` that will be needed for the pull request.
```

Comment:
Same comment as above re: `recipe.py` not being the required name for the recipe module. Not sure the best way to handle this, maybe just

```
It has a template for the recipe module and `meta.yaml` that will be needed for the pull request.
```

Note also that for consistency across feedstocks, we currently don't accept the short form `.yml` extension, only `.yaml`.